apps/webpack.js

```diff
@@ -1,7 +1,12 @@
 var _ = require('lodash');
 var webpack = require('webpack');
 var path = require('path');
-var config = {
+var LiveReloadPlugin = require('webpack-livereload-plugin');
+
+var AUTO_RELOAD = ['true', '1'].indexOf(process.env.AUTO_RELOAD) !== -1;
```

Comment:
I would say either a helper method (perhaps https://github.com/ctavan/node-getenv?), or a shared module that explicitly exports all the environment variables the app uses (with appropriate type casts), and even documents those environment variables. The latter would be nice in that it would make conflicts really obvious.

```javascript
module.exports = {
  AUTO_RELOAD: getBoolEnv('AUTO_RELOAD'),
  DEV: getBoolEnv('AUTO_RELOAD'),
  ...
}